src/main/resources/application.yml

```diff
@@ -126,14 +128,45 @@ nhais:
       +yLvICQmVjHWsNWD3uuT3gEGdeKDIzEnovXUmqhQE7N/YDupTv5hkh8hMKl12XGs
       3WJ4bVSmp3MOQ2EF4zrAl4fXEd4lS3JxKtzbHFbSogAmPFnuP8wIDWTDgfEq
       -----END RSA PRIVATE KEY-----}
-    cypherToMailbox: ${NHAIS_MESH_CYPHER_TO_MAILBOX:XX11=nhais_mailbox
-      TES5=nhais_mailbox
-      YY21=nhais_mailbox
-      KC01=nhais_mailbox
-      WHI5=nhais_mailbox
-      SO01=nhais_mailbox
-      UNY5=nhais_mailbox}
-    scanMailboxIntervalInMilliseconds: ${NHAIS_SCAN_MAILBOX_INTERVAL_IN_MILLISECONDS:60000}
-    scanMailboxDelayInSeconds: ${NHAIS_SCAN_MAILBOX_DELAY_IN_SECONDS:300}
+    subCAcert: ${NHAIS_MESH_SUB_CA:-----BEGIN CERTIFICATE-----
+      MIIFXzCCA0egAwIBAgIJALRbCSor9bEbMA0GCSqGSIb3DQEBCwUAMEUxCzAJBgNV
+      BAYTAkFVMRMwEQYDVQQIDApTb21lLVN0YXRlMSEwHwYDVQQKDBhJbnRlcm5ldCBX
+      aWRnaXRzIFB0eSBMdGQwIBcNMTYwNDIwMTczOTA5WhgPMjIxNjAzMDMxNzM5MDla
+      MEUxCzAJBgNVBAYTAkFVMRMwEQYDVQQIDApTb21lLVN0YXRlMSEwHwYDVQQKDBhJ
+      bnRlcm5ldCBXaWRnaXRzIFB0eSBMdGQwggIiMA0GCSqGSIb3DQEBAQUAA4ICDwAw
+      ggIKAoICAQDTHtXZykS5dsvFvhMWKB43p79NWJfHgH0sy4fDAQ1fU/lXc7kzR398
+      sIWu67l1Tc9YqhTatPNSR+WPlc7O7bEB+1iLcoMzNFHawHLGOPFv+owjJsY3XMLV
+      rOUPOpzT+ggaieI+FBxx0PIDfleRiTdkAtozC5wKDxEJtrpt4blMMit7VaeTp7dX
+      ryJv1VAq034wjLzLjBI2l9H7Gx+JVCGtrxTxwDZrpoIdkJ7uCxOdO3bwpT2fvR8O
+      RwmefC90q25sabeIodJxu2iqqRawHtaQ7g4WEimK1ed4hUoyOkxwtjfYV26vRXUP
+      vJlTuoh3FhHZw5xAODBt+fMqJFjPlAYfCtMkXVNflu5ObTrQ5iQme40Ugbh3rSNE
+      fQV68D6Qs9Bfs9EWMBx3h8PsLg7WCbaKhcm0oMavHXr12lH/z/nCf+cU+61U1yJ+
+      Bn9g2Fn8iEb7b4J3qpQ+Cxf/pYknrXsgYY4p8j0EkQgBZDoHr7Tgb+koUOcn1InK
+      NQLX2KV7MTe9P7ikn/kbE+HpW5bCL+N2KETV9qlqJ0mUR2H5+eaX+yme2Af0afFZ
+      Z9rb97jzC9Elad3F6NK7Jdr6u3cPKIaU3qbuYDq3MxdtMGjiWyYRzixofpMgUqYm
+      8EEv6GsT+eZNY+rhlyUkEou6upgOkLmZcNsbEM/K+JJm0bW87YoVBwIDAQABo1Aw
+      TjAdBgNVHQ4EFgQUXxJ+6UmJ4cVNqXurwQJeyQALP4YwHwYDVR0jBBgwFoAUXxJ+
+      6UmJ4cVNqXurwQJeyQALP4YwDAYDVR0TBAUwAwEB/zANBgkqhkiG9w0BAQsFAAOC
+      AgEAmrJdePkitQtZmhopjnw11eT0WmvZnZfPDccm8LJ8uevQfHKdM5uruDGeE0r5
+      SbUGuxC08Bs42ZNnqbXsbSzK7piHBfwG95VYC9knfBBg2Gv9EDKSJnr8NXdDyJeF
+      caycrkFeGkdRal3bCfXO3rHQiMtukFG+x/OzD9gjEPEfmbBXC/gOesWJNFmS4doG
+      +A4lAIXEbvDYdlzH/jh9C26t8LWwRirL9Hn6fHQbN4ywDH4n3FRvSElwH/uk/mEv
+      4ksPphrAV760Rxsjy6XM2+IwFs1BrQVkPKT91WirFampmvsKModqhXJPb0SPLdd0
+      G4sfyeiRzbzDjiq4igvD1HLIulPrbkdpLNUOsaSL56xE19FpHjl95OI9leKQeUMd
+      No3A+gh00sBCYZvhHXCPVACQGOke8JeChxBmqWWuOdBXyXyE9iSZtN4SqNhWjGTQ
+      97cqxZ4zgzmvb5FaGyG6ZmlWjCpkwNpbETp6CmqqN4sr//QnY+fc/6k5E3V31JWu
+      lB6mv/JlORiEnDZjP6fO1L7yEvvc1vMlMMDCfNLbU5sdmp3LouR1Yf7uMzGzVL8r
+      MsL1ds0kLcY9oQlduXOumM/bj+0hJTHxc06tr9ilzDAgQ+m48+xLxa6+X27m5pnl
+      W/JNIRmhLoeFNGNh8HvhI2PwOCsFiqT1rrCaUtusTyH0Ggs=
+      -----END CERTIFICATE-----}
+    cypherToMailbox: ${NHAIS_MESH_CYPHER_TO_MAILBOX:XX11=A91562OT001
+      TES5=A91562OT001
```

Comment:
Looks like your mailbox ID, revert to nhais_mailbox?

Reply:
reverted